apitests/gulpfile.js

```diff
@@ -38,7 +38,9 @@ gulp.task('build', 'Runs build:clean, tsd and ts in sequence', function (callbac
 });
 
 gulp.task('test-jasmine', 'Starts jasmine to run tests', (done) => {
-    return gulp.src(['dist/**/*.js'])
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;
+
```

Comment:
It looks like for some reason jasmine-gulp isn't loading up the helpers, that's why we moved it inline to gulp. If there's a way to get jasmine-gulp to exec the helpers then agree, it should remain in helpers